### PR TITLE
Task 1.1 BobCatC

### PR DESCRIPTION
**Pull request by BobCatC**
В задании был реализован алгоритм KNN. 
После этого скачал данные Street View House Numbers и провел обучение. 
С помощью cross-validation был подобран параметр k - рассматриваемое количество соседей, равное 10.
Точность составила 0.21 на тестовой выборке.

### CODE REVIEW

**Comment by BobCatC** [comment]
Хотелось бы заметить, что в кросс-валидации входные массивы случайно сортируются, поэтому наилучшее значение параметра **k**, увиденное мной, может отличаться от полученных вами.

**Comment by BobCatC** [comment]
Завершил задание 1.2
Реализовал линейную классификацию с использованием Softmax, Cross Entropy Loss и SGD.
После проведения кросс валидации были найдены наилучшие значение гиперпараметров _learning_rate = 0.02_ и _reg_strength = 1e-4_. Точность на валидационном датасете составила 0.201.
